env.go

```diff
@@ -37,9 +37,9 @@ func Set(name string) {
 	Env = name
 }
 
-// IsProd returns true if the current environment is `production` or `prod`.
+// IsProd returns true if the current environment is `production` or `prod` or `divert`.
 func IsProd() bool {
-	return Env == "production" || Env == "prod"
+	return Env == "production" || Env == "prod" || Env == "divert"
```

Comment:
Wow. I was not even aware that we need a whole package to deal just with one environment variable.

I think the idea long time ago was to have smth like https://github.com/bkeepers/dotenv .

The downside of this pr is that is adds some custom Env specific for Remerge - `divert` - but it is a public repo actually =)

I think we need to make it private and in the long term maybe shut it down completely - we do not use it as intended.

WDYT @itszootime ?

Reply:
If there will be no response - merge it, I have approved it.
I think it is not critical anyway